kloppy/infra/serializers/event/wyscout/deserializer_v2.py

```diff
@@ -103,13 +103,7 @@ def _parse_shot(raw_event: Dict, next_event: Dict) -> Dict:
 
     if next_event["eventId"] == wyscout_events.SAVE.EVENT:
         if next_event["subEventId"] == wyscout_events.SAVE.REFLEXES:
-            qualifiers.append(
-                GoalkeeperActionQualifier(GoalkeeperAction.REFLEX)
-            )
-        if next_event["subEventId"] == wyscout_events.SAVE.SAVE_ATTEMPT:
```

Comment:
Not sure I understand the removal of the `SAVE_ATTEMPT` here. Does it never happen as it only happens on `wyscout_events.SAVE.EVENT` ?